src/kotlin_editor_export_plugin.cpp

```diff
@@ -143,4 +146,36 @@ String KotlinEditorExportPlugin::get_name() const {
     return "Godot Kotlin/Jvm";
 }
 
+bool KotlinEditorExportPlugin::_begin_customize_resources(const Ref<EditorExportPlatform>& p_platform, const Vector<String>& p_features) {
+    return true;
+}
+
+uint64_t KotlinEditorExportPlugin::_get_customization_configuration_hash() const {
+    // Mandatory to implement when customizing resources. The hash is used to keep separate configuration depending on the export options.
+    // We simply return a constant as source files are going to be cleared regardless of the configuration.
+    return 0;
+}
+
+Ref<Resource> KotlinEditorExportPlugin::_customize_resource(const Ref<Resource>& p_resource, const String& p_path) {
+    String ext = p_path.get_extension();
+
+    // We create a new resource, otherwise it overwrites the one in the project, not just in the export.
+    // After export, source files are only used to find the right script in the .jar using its path or name.
+    if (ext == GODOT_JVM_REGISTRATION_FILE_EXTENSION) {
```

Comment:
Do we really need to clean those as this is not source code ?

Reply:
We don't really need to, they only contain the list of properties, method and signal, which is information easily obtainable from the Script API. But its content also has no utility at all for running the game, so I just thought I would be thorough and clean all 3 kinds of scripts equally.